pkgs/development/python-modules/finetuning-scheduler/default.nix

```diff
@@ -29,6 +30,13 @@ buildPythonPackage rec {
     hash = "sha256-neeSATQwAaYN1QGBUXphqqJp9lP3HG2OH4aLdt1cOho=";
   };
 
+  patches = [
+    (fetchpatch2 {
+      url = "https://github.com/speediedan/finetuning-scheduler/commit/78e6e225f353d1ba95db05d7fc6ff541859ed6a2.patch?full_index=1";
+      hash = "sha256-Um7Tsps+LzOD8nBY5h+Eq6nsfgefUeff2eK8b+doWfc=";
+    })
```

Comment:
```suggestion
    (fetchpatch {
      url = "https://github.com/speediedan/finetuning-scheduler/commit/78e6e225f353d1ba95db05d7fc6ff541859ed6a2.patch";
      hash = "sha256-Um7Tsps+LzOD8nBY5h+Eq6nsfgefUeff2eK8b+doWfc=";
    })
```

Reply:
@prusnak Is there some guidance explaining when one should use `fetchpatch` vs `fetchpatch2`? I'm genuinely curious, since docs don't see to explain it, and I've seen both forms used. Should I always prefer `fetchpatch`?